src/DacpacTool/Program.cs

```diff
@@ -197,7 +198,14 @@ private static int DeployDacpac(DeployOptions options)
                     }
                 }
 
-                deployer.UseTargetServer(options.TargetServerName);
+                if (options.TargetPort != null)
```

Comment:
Perhaps more idiomatic C# would be to do `if (options.TargetPort.HasValue)` here ;).

Reply:
Made that change, I believe we should be good to merge this?